slurp/src/main/webapp/index.html

```diff
@@ -2,17 +2,21 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
-    <link rel="stylesheet" href="style.css">
-    <script src="script.js"></script>
+    <title>Slurp</title>
+    <!-- CSS -->
+    <link rel="stylesheet" href="/styles/style.css">
+
+    <!-- JavaScript -->
+    <!-- The core Firebase JS SDK is always required and must be listed first -->
+    <script defer src="https://www.gstatic.com/firebasejs/7.15.4/firebase-app.js"></script>
+    <!-- Add any other SDKs for Firebase products that we want to use
+        https://firebase.google.com/docs/web/setup#available-libraries -->
+    <script defer src="https://www.gstatic.com/firebasejs/7.15.4/firebase-firestore.js"></script>
+    <script defer src="https://www.gstatic.com/firebasejs/7.15.4/firebase-auth.js"></script>
+
+    <script defer src="/scripts/init-firebase.js"></script>
+    <script defer src="/scripts/script.js"></script>
```

Comment:
```suggestion
    <script defer src="scripts/init-firebase.js"></script>
    <script defer src="scripts/script.js"></script>
```

These links should be relative.